File: Compass Demo/app/src/main/java/com/example/compassdemo/MainActivity.java
package com.example.compassdemo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void gotoCompass(View view) {
        startActivity(new Intent(MainActivity.this, CompassActivity.class).putExtra("lat", 23.745002).putExtra("lng", 90.355151));
    }
}

File: Android Official Kotlin Basic with Fundamentals/app/src/main/java/com/technobd/basickotlinprogramming/MainActivity.kt
package com.technobd.basickotlinprogramming

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // calling the printSomething method here.
        printSomething()

    }

    fun printSomething(){
        println("Happy Birthday!")
        println("Jhansi")
        println("You are 25!")
    }
}